app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/codePager/SampleInfoScreen.kt

```diff
@@ -38,31 +32,31 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
-import androidx.compose.ui.input.nestedscroll.nestedScroll
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import coil3.compose.AsyncImage
 import com.esri.arcgismaps.kotlin.sampleviewer.R
 import com.esri.arcgismaps.kotlin.sampleviewer.model.DefaultSampleInfoRepository
 import com.esri.arcgismaps.kotlin.sampleviewer.model.Sample
 import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.CodeView
 import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.ReadmeView
+import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.SampleViewerTopAppBar
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
 
 /**
  * Shows both README and Code for each sample.
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun SampleInfoScreen(
     onBackPressed: () -> Unit,
     sampleName: String,
     optionPosition: Int
 ) {
+    // TODO: Can we pass in this sample as an argument?
+    //  The preview does not render due this call
     val sampleData = DefaultSampleInfoRepository.getSampleByName(sampleName)
```

Comment:
Sure, when I mocked up the code it works just fine!

Just need to make this call `DefaultSampleInfoRepository.getSampleByName(sampleName)` in NavGraph.kt and pass the sample as the argument rather than the sample name